docs/dagster-university/pages/dagster-dbt/lesson-3/5-loading-dbt-models-into-dagster-as-assets.md

```diff
@@ -28,32 +28,26 @@ We’ll only create one `@dbt_assets` definition for now, but in a later lesson,
 
 ## Loading the models as assets
 
-1. Create a new file in the `assets` directory called `dbt.py`.
+1. Navigate to the `assets/dbt.py` created earlier and open it.
 
 2. Add the following imports to the top of the file:
 
    ```python
    from dagster import AssetExecutionContext
    from dagster_dbt import dbt_assets, DbtCliResource
-
-   import os
-
-   from .constants import DBT_DIRECTORY
    ```
 
-3. The `@dbt_assets` decorator requires a path to the project’s manifest file, which is within our `DBT_DIRECTORY`. Use that constant to create a path to the `manifest.json` by copying and pasting the code below:
+3. The `@dbt_assets` decorator requires a path to the project’s manifest file, which is within our `dbt_project` representation created earlier. The manifest path can be easily accessed with the code below:
 
    ```python
-   dbt_manifest_path = os.path.join(DBT_DIRECTORY, "target", "manifest.json")
+   dbt_project.manifest_path
    ```
 
-   Similar to how we used `joinpath` earlier to point to the dbt project’s directory, we’re using it once again to reference `target/manifest.json` more precisely.
-
 4. Now, use the `@dbt_assets` decorator to create a new asset function and provide it with a reference to the manifest:
 
    ```python
    @dbt_assets(
-       manifest=dbt_manifest_path,
+       manifest=dbt_project.manifest_path,
    )
    def dbt_analytics(context: AssetExecutionContext, dbt: DbtCliResource):
    ```
```

Comment:
Step 3 strikes me as a little odd when I look at what Step 4 says. Here, it doesn't look like we're actually instructing the user to do anything - I was a little confused and left wondering what I was supposed to do with the code.

What if we combined Steps 3 and 4 to do something like the following? Feel free to wordsmith:

---

3. Next, we'll use the `@dbt_assets` decorator to create a new asset function and provide it with a reference to the project's manifest file:

   ```python
   @dbt_assets(
       manifest=dbt_project.manifest_path,
   )
   def dbt_analytics(context: AssetExecutionContext, dbt: DbtCliResource):
   ```

   Here, we used `dbt_project.manifest_path` to provide the reference to the project's manifest file. This is possible because the `dbt_project` representation we created earlier contains the manifest path, accessible by using the `manifest_path` attribute.